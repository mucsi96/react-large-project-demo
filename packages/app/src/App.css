body {
  margin: 20px 100px;
}
