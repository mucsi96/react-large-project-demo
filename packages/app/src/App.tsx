import React, { FC } from 'react';
import { FriendsList } from 'friends';
import './App.css';

const App: FC = () => <FriendsList />;

export default App;
