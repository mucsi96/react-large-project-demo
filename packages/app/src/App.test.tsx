import { shallow } from "enzyme";
import React from "react";
import App from "./App";

test("renders learn react link", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});
