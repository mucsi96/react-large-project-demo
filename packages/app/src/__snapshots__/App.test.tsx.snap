// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders learn react link 1`] = `
<div
  className="App"
>
  <header
    className="App-header"
  >
    <img
      alt="logo"
      className="App-logo"
      src="logo.svg"
    />
    <p>
      Edit 
      <code>
        src/App.tsx
      </code>
       and save to reload.
    </p>
    <a
      className="App-link"
      href="https://reactjs.org"
      rel="noopener noreferrer"
      target="_blank"
    >
      <Button>
        Learn React
      </Button>
    </a>
  </header>
</div>
`;
